### PR TITLE
[SYCLomatic] Implement migration for {CUB_MIN, CUB_MAX, CUB_RUNTIME_FUNCTION}

### PR DESCRIPTION
**Pull request by yihwang-nv**

### CODE REVIEW

File: clang/lib/DPCT/MapNames.cpp [hunk 1]
```diff
@@ -4396,7 +4396,14 @@ std::unordered_map<std::string, MacroMigrationRule> MapNames::MacroRuleMap{
                         "CU_LAUNCH_PARAM_END", "((void *) 0)",
                         HelperFeatureEnum::device_ext)},
     {"CUDART_PI_F", MacroMigrationRule("CUDART_PI_F", RulePriority::Fallback,
-                        "CUDART_PI_F", "3.141592654F")}
+                                       "CUDART_PI_F", "3.141592654F")},
+    {"CUB_MAX", MacroMigrationRule("cub_macro_rule", RulePriority::Fallback,
+                                   "CUB_MAX", "sycl::max")},
```
Comment (tangjj11) [inline clang/lib/DPCT/MapNames.cpp]: should use `getClNamespace`?

**Review by tangjj11 — CHANGES_REQUESTED** [review]

File: clang/lib/DPCT/MapNames.cpp (hunk 1 above)
Comment (yihwang-nv) [reply clang/lib/DPCT/MapNames.cpp]: done

**Review by tangjj11 — APPROVED** [review]

**Review by AndyCHHuang — APPROVED** [review]
LGTM

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**